app/gui/language/ast/impl/src/crumbs.rs

```diff
@@ -129,17 +129,17 @@ struct NonChildTreeCrumb;
 ///
 /// It provides way to easily convert vector of specific crumbs (e.g.
 /// `[InfixCrumb::LeftoOperand, ..]` without calling into on each element.
-pub trait IntoCrumbs: IntoIterator<Item: Into<Crumb>> + Sized {
+pub trait IntoCrumbs<I: Into<Crumb>>: IntoIterator<Item = I> + Sized {
```

Comment:
I don't understand why this change is needed. If anyone can fix the old code – you're welcome.

Reply:
Thank you, done